src/mon/PGMap.cc

```diff
@@ -1367,6 +1367,8 @@ void PGMap::dump(Formatter *f) const
   dump_pg_stats(f, false);
   dump_pool_stats(f);
   dump_osd_stats(f);
+  pg_sum_delta.dump(f);
```

Comment:
we already have `PGMapDigest::overall_client_io_rate_summary` function, we could achieve `iostats` command via adding a python_get('client_io`) interface. likes https://github.com/ceph/ceph/pull/19985/commits/6272c2636425d98b3194828157366527977277bb

Reply:
yes, `pg_sum_delta` (dumped here) is what is indeed used by `PGMapDigest::overall_client_io_rate_summary`. I added a 'io_rate' as a way to fetch it from a mgr plugin.